dart/lib/src/protocol/debug_image.dart

```diff
@@ -57,9 +57,7 @@ class DebugImage {
       json['uuid'] = uuid;
     }
 
-    if (type != null) {
-      json['type'] = type;
-    }
+    json['type'] = type;
```

Comment:
type is always required https://develop.sentry.dev/sdk/event-payloads/debugmeta#debug-images
I guess we dont need to check the other fields, the server will just ignore the debug image if its incomplete, better than checking on each SDK, and this might change over time